emcc.py

```diff
@@ -412,9 +409,14 @@ def standardize_setting_change(key, value):
     if key in shared.Settings.internal_settings:
       exit_with_error('%s is an internal setting and cannot be set from command line', key)
 
+    if key in shared.Settings.legacy_settings and key in shared.Settings.alt_names:
+      actual_key = shared.Settings.alt_names[key]
+    else:
+      actual_key = key
```

Comment:
with this approach, we don't set the actual key's value, but the legacy key, as lower down we set using `key`. is that intentional? it seems riskier than the old approach unless I'm missing something.

Reply:
Yes, this check was redundant.  The handling the legacy keys and the handling of reporting errors about legacy key usage is all done in the `Settings.__setattr__`.  If we rename the settings here never see the warning.